apps/fft/Makefile

```diff
@@ -25,5 +25,18 @@ bench_48x48: bench_fft
 bench_64x64: bench_fft
 	./bench_fft 64 64
 
+# Generate four AOT compiled FFT variants. Forward versions have gain set to 1 / 256.0
```

Comment:
This reinforces a separate issue: we need to come up with a less-cumbersome way to use Generators with Make. (Or find a better build system.)